### DISCUSSION TITLE
Build for android 7.1.2

### DISCUSSION THREAD

**Discussion by RosiersRobin** [rustdesk/rustdesk#7376] [Q&A]
Hi,

We have older machines running android 7.1.2, I'd like to have headless access to those machines. When I try to install the latest apk version, it fails to do so. I'm getting the error 'could not install app'. Any ideas?

**Comment by xlionjuan** [comment]
Too old.

**Comment by RosiersRobin** [comment]
Using the nightly build it installs just fine though. ;D
